openpype/hosts/houdini/plugins/create/create_pointcache.py

```diff
@@ -83,6 +83,7 @@ def create(self, subset_name, instance_data, pre_create_data):
     def get_network_categories(self):
         return [
             hou.ropNodeTypeCategory(),
+            hou.objNodeTypeCategory(),
```

Comment:
I actually think the current implementation of `pointcache` publishing and its validation does not allow publishing from Object level. Validations will need to be altered + the configuration of the ROP should be different too.

Reply:
The current implementation gets the sop node if an obj node was selected. 
I agree with you the behavior should be altered so that when selecting an OBj mode it uses the obj options in the Alembic ROP as mentioned here https://github.com/ynput/OpenPype/issues/5290

Reply:
That'd still be weird having that ROP node at /obj level itself then? I'm not sure adding this now while OBJ level exports itself are not supported is the best way forward - but I'm not too bothered by it so will leave it up to you.